### PR TITLE
feat(STN-237):  qa feedback

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
Please see [JIRA card](https://sparkbox.atlassian.net/browse/STN-237) for issues to check for in this PR.

## Steps to Test
1. For the local footer, you'll need to add a new font awesome icon that is not linked to the footer. For example, you can use a WYSIWYG block, to add a link, and then beside the link, you could insert the font awesome icon. Ensure that the font awesome icon does not have a hover state when placed like this.
2. Make sure your node news detail page has no sidebar and check the two issues there.
3. Run npm run test on the humsci_basic folder and make sure tests and linting pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss [hunk 1]
```diff
@@ -115,20 +115,7 @@ button:not([class]) {
...
-      background: svg('<svg viewBox="0 0 12 12"><path d="M11.273 5.818v5.455H.727V.727h5.091m1.455 0h4v4m0-4L6.182 5.818" fill="transparent" stroke="#{$hb-lighten-color--primary}" stroke-width="2" stroke-miterlimit="10"/></svg>') center center no-repeat;
-      height: hb-calculate-rems(12px);
-      width: hb-calculate-rems(12px);
-      display: inline-block;
-      vertical-align: middle;
-      position: absolute;
-      top: hb-calculate-rems(4px);
-      right: hb-calculate-rems(-18px);
-    }
+    @include hb-external-link-icon;
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss:118]: Nice refactor. 🎉

**Review by bryanbraun — APPROVED** [review]
The changes look great. I'm approving what's here, but I also think this is an opportunity to be more thoughtful about how we are styling font-awesome icons. Here's what I think some better default behaviors could be:

**Proposed Font Awesome Icon behaviors:**

- All font awesome icons inherit their color from the theme (and color pairing, if applicable). That color is consistent, wherever on the site the icon is placed, and isn't configurable by end-users.
- Font awesome styles (solid, regular, etc),  and sizes (1x, 2x, etc), should be fully customizable via the WYSIWYG options.
- Font awesome icons can be linked, by entering the "source" view of the WYSIWYG editor, and manually converting the `<span>` to an `<a href="">`. These linked icons would adopt hover states.

---

Currently these behaviors aren't applied consistently, so I'd like to push up a commit that fixes this. @silverli , please hold on merging until I've pushed up this commit, and you've confirmed the fix.

**Comment by bryanbraun** [comment]
I've commited the font-awesome fixes.



<details>
    <summary>If you want some example markup for testing, here's what I was using:</summary>
<pre>

&#x3C;p&#x3E;Unlinked Icons:&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-twitter fab&#x22;&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fa-fw fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-fw fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-fw fa-inverse fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fa-fw fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-border fa-fw fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-fw fa-inverse fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fa-border fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-border fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E; &#x3C;span class=&#x22;fa-border fa-inverse fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-twitch fab&#x22;&#x3E;&#x3C;/span&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-envelope fas&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-envelope far&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-envelope fa-pull-left fas&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x3C;/span&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;Linked Icons:&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;a class=&#x22;fab fa-facebook-square&#x22; href=&#x22;http://facebook.com&#x22;&#x3E;&#x3C;span class=&#x22;visually-hidden&#x22;&#x3E;Facebook&#x3C;/span&#x3E;&#x3C;/a&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;a class=&#x22;fa-2x fa-envelope fas&#x22; href=&#x22;#&#x22;&#x3E;&#x3C;span class=&#x22;visually-hidden&#x22;&#x3E;Envelope&#x3C;/span&#x3E;&#x3C;/a&#x3E; &#x3C;a class=&#x22;fa-2x fa-envelope far&#x22; href=&#x22;#&#x22;&#x3E;&#x3C;span class=&#x22;visually-hidden&#x22;&#x3E;Envelope&#x3C;/span&#x3E;&#x3C;/a&#x3E; &#x3C;a class=&#x22;fa-envelope fal&#x22; href=&#x22;#&#x22;&#x3E;&#x3C;span class=&#x22;visually-hidden&#x22;&#x3E;Envelope&#x3C;/span&#x3E;&#x3C;/a&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;Multiple Sizes:&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;/span&#x3E;&#x3C;span&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-envelope-open-text fas&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-link fas&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-2x fa-link fas&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x3C;/span&#x3E;&#x3C;/p&#x3E;

&#x3C;p&#x3E;&#x3C;span class=&#x22;fontawesome-icon-inline&#x22;&#x3E;&#x3C;span class=&#x22;fa-3x fa-twitter fab&#x22;&#x3E;&#x3C;/span&#x3E;&#x26;nbsp;&#x3C;/span&#x3E;&#x3C;/p&#x3E;


</pre>
</details>

I also committed the compiled css because I figured we needed to do that before merging.

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss [hunk 2]
```diff
@@ -1,22 +1,49 @@
-// Font Awesome Icons
+// Principles for how theme styles interact with font-awesome icons:
+//
+// - Font awesome icons inherit their color from the theme (and color pairing,
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss:3]: 👍 great docs!

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss [hunk 3]
```diff
@@ -1,18 +1,49 @@
...
+  &.fas,
+  > .fa,
+  > .fab,
+  > .fal,
+  > .far,
+  > .fas {
+    text-decoration: none;
+
+    @include hb-colorful {
+      &:hover {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss]: can you add :focus here too?

Comment (bryanbraun) [reply docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss]: Updated! Good catch. 👍

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss [hunk 4]
```diff
@@ -1,18 +1,50 @@
...
+  > .fa,
+  > .fab,
+  > .fal,
+  > .far,
+  > .fas {
+    text-decoration: none;
+
+    @include hb-colorful {
+      &:hover,
+      &:focus {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss:40]: looks great! I approve.

**Pull request merged**